<!--REQUEST_ID=258e1c54-cd1d-45b2-b176-020091697ad2-->
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48715&attempt_id=69923b89-8e96-4272-8d38-c853e66fd23e)

Attempted: firefoxPackages.tor-browser

<details><summary>Partial log (click to expand)</summary><p>

```
 0:07.07   File "/build/tor-browser/python/mozbuild/mozbuild/configure/__init__.py", line 1003, in wrapped
 0:07.07     return new_func(*args, **kwargs)
 0:07.07   File "/build/tor-browser/build/moz.configure/init.configure", line 580, in split_triplet
 0:07.08     cpu, manufacturer, os = triplet.split('-', 2)
 0:07.08 ValueError: need more than 2 values to unpack
 0:07.10 *** Fix above errors and then restart with\
 0:07.10                "/nix/store/fabhw7nzi5ij0ff3dljxnfnldgzrcqgv-gnumake-4.2.1/bin/make -f client.mk build"
 0:07.10 make: *** [client.mk:149: configure] Error 1
builder for '/nix/store/4im5z6m9wa9hyc3jr60w7cc18cp0a8i5-tor-browser-unwrapped-8.0.2.drv' failed with exit code 2
error: build of '/nix/store/4im5z6m9wa9hyc3jr60w7cc18cp0a8i5-tor-browser-unwrapped-8.0.2.drv' failed
```
</p></details>

--- a/pkgs/applications/networking/browsers/firefox/packages.nix
+++ b/pkgs/applications/networking/browsers/firefox/packages.nix
@@ -9,9 +9,15 @@ let
   ];
 
   firefox60_aarch64_skia_patch = fetchpatch {
-      name = "aarch64-skia.patch";
-      url = https://src.fedoraproject.org/rpms/firefox/raw/8cff86d95da3190272d1beddd45b41de3148f8ef/f/build-aarch64-skia.patch;
-      sha256 = "11acb0ms4jrswp7268nm2p8g8l4lv8zc666a5bqjbb09x9k6b78k";
+    name = "aarch64-skia.patch";
+    url = https://src.fedoraproject.org/rpms/firefox/raw/8cff86d95da3190272d1beddd45b41de3148f8ef/f/build-aarch64-skia.patch;
+    sha256 = "11acb0ms4jrswp7268nm2p8g8l4lv8zc666a5bqjbb09x9k6b78k";
+  };
+
+  firefox60_triplet_patch = fetchpatch {
+    name = "triplet.patch";
+    url = https://hg.mozilla.org/releases/mozilla-release/raw-rev/bc651d3d910c;
+    sha256 = "0iybkadsgsf6a3pq3jh8z1p110vmpkih8i35jfj8micdkhxzi89g";
   };
 
 in
@@ -110,7 +116,8 @@ rec {
       find . -exec touch -d'2010-01-01 00:00' {} \;
     '';
 
-    patches = nixpkgsPatches;
+    patches = nixpkgsPatches
+      ++ lib.optional (args.tbversion == "8.0.2") firefox60_triplet_patch;
 
     meta = {
       description = "A web browser built from TorBrowser source tree";

